src/components/atomic/audio-player/audio-player.jsx

```diff
@@ -59,7 +60,9 @@ const AudioPlayer = ({ audioSrc, lang = 'en', tracking = () => {} }) => {
 					paused ? 'paused' : ''
 				}${hasError ? 'error' : ''}`}
 				onClick={handlePlay}>
-				<span className="show-for-sr">{srText}</span>
+				<span data-testid="span.show-for-sr" className="show-for-sr">
```

Comment:
weird name for a testid

Reply:
I was able to remove test ids from the audio-player and the video-player. Some of the other ones are too embedded.